src/Main.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class Main {
+    static Scanner keyboard = new Scanner(System.in);
+
+    public static void chekYear(int year) {
+        if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
+            System.out.println(year + " год -  високосный год");
+        } else {
+            System.out.println(year + " год - невисокосный год");
+        }
+    }
+
+    public static void chekPhone() {
```

Comment:
```suggestion
    public static void printSuggestionForInstallation() {
```
название метода должно отражать то, что он делает

Reply:
метод должен принимать 2 параметра (см. задание)